tests/Application/config/packages/twig_hooks.yaml

```diff
@@ -16,6 +16,88 @@ sylius_twig_hooks:
             app_javascripts:
                 template: "shop/javascripts.html.twig"
 
+        'bitbag_sylius_wishlist_plugin.wishlist.index':
```

Comment:
Everything from here down should be in the main app config, instead of the test app 💃